src/components/Metadata/index.jsx

```diff
@@ -29,7 +29,7 @@ const Metadata = ({ metadata, handleChange }) => {
             type="text"
             name="title"
             id="title"
-            value={metadata.title || metadata.name}
```

Comment:
@risenW I think the `metadata.name` property is important here, because if some resource doesn't contain the title property will render the name.